go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/twpayne/chezmoi/v2
 
-go 1.22.7
-
-toolchain go1.23.3
+go 1.23.4
```

Comment:
I noticed that github.com/tobischo/gokeepasslib/v3 requires Go 1.23. FYI I'm working on a separate PR to switch chezmoi over to Go 1.23. I'll ask you to rebase this PR when its done.